samples/OpenIdConnectSample/hosting.json

```diff
@@ -0,0 +1,4 @@
+﻿{
+  "server": "Microsoft.AspNet.Server.Kestrel",
+  "server.urls": "http://localhost:42023"
```

Comment:
This can't be specified both here and in the IIS express launch settings, it will conflict at runtime. Move it to an environment variable in the web profile.